src/app/components/UserAssets/VestedAssets.tsx

```diff
@@ -181,12 +185,28 @@ function AssetRow({
         />
       </td>
       <td className="tw-text-right">
-        <ActionButton
-          className="tw-inline-block"
-          text={t(translations.userAssets.actions.withdraw)}
-          disabled={contract === ethGenesisAddress || loading}
-          onClick={() => onWithdraw(contract)}
-        />
+        {withdrawLocked && contract !== ethGenesisAddress ? (
```

Comment:
Not sure if you need this `contract !== ethGenesisAddress` here, contract == ethGenesisAddess when response to request is not yet received from blockchain